src/Tools/ExternalAccess/CodeLens/Microsoft.CodeAnalysis.ExternalAccess.CodeLens.csproj

```diff
@@ -0,0 +1,37 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Library</OutputType>
+    <RootNamespace>Microsoft.CodeAnalysis.ExternalAccess.CodeLens</RootNamespace>
+    <TargetFramework>net472</TargetFramework>
+
+    <!-- NuGet -->
+    <IsPackable>true</IsPackable>
+    <PackageId>Microsoft.CodeAnalysis.ExternalAccess.CodeLens</PackageId>
+    <PackageDescription>
+      A supporting package for CodeLens:
+      [Link needed]
+    </PackageDescription>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <!--
+      ⚠ ONLY CODELENS ASSEMBLIES MAY BE ADDED HERE ⚠
```

Comment:
This stuff builds directly in the VS repo, doesn't it? We're also planning on doing ExternalAccess libraries for those? I thought we weren't, but would understand why you would want to. This is more a "if we're changing the plan, just want to know that we are changing the plan" question.

Reply:
This is generalized by the build boss rules. The new approach applies to everything which:

1. Does not build in dotnet/roslyn
2. Ships with or loads within Visual Studio process(es)

These rules ensure that PR builds of Roslyn are correct even before insertion-time checks.